src/test/java/ci.sql

```diff
@@ -247,5 +247,10 @@ CREATE TABLE `test_partition_aggregation` (
     `c2` bigint DEFAULT NULL,
     PRIMARY KEY (`c1`))partition by range(`c1`)(partition p0 values less than(200), partition p1 values less than(500), partition p2 values less than(900));
 
+CREATE TABLE test_aggregation_with_filter (
```

Comment:
Try to use test_partition_aggregation for tests with filters.

Reply:
fixed